src/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationReader.cs

```diff
@@ -111,7 +111,7 @@ public abstract class XmlSerializationReader : XmlSerializationGeneratedCode
         private string _guidID;
         private string _timeSpanID;
 
-        private static bool s_checkDeserializeAdvances;
+        private static bool s_checkDeserializeAdvances=false;
```

Comment:
Who changes the value of this field?

Reply:
@stephentoub, good point. Looks like this was used in full framework, but not anymore on core as the configuration is not supported.

http://referencesource.microsoft.com/#System.Xml/System/Xml/Serialization/XmlSerializationReader.cs,123

@yujayee, I think we can clean up this variable and related code in this case.